src/app/components/team/Similarity/SimilarityComponent.js

```diff
@@ -274,6 +274,23 @@ const SimilarityComponent = ({
                     label="Elasticsearch suggestion threshold"
                     error={(settings.text_elasticsearch_suggestion_threshold > settings.text_elasticsearch_matching_threshold)}
                   />
+                  <Box ml={7}>
+                    <Typography component="span" style={{ fontWeight: 'bold' }} className={classes.transactionMargin}>
+                      <FormattedMessage
+                        id="similarityComponent.textLength"
+                        defaultMessage="Minimum words required for a confirmed match"
+                      />
```

Comment:
Please add a `description` field for localization. In this case it could be

`description="A label on a text input where the user specifies the minimum number of words needed to match before a text content match is considered confirmed"`

Reply:
Done

Reply:
Thank you!